src/ui/components/sdcard.c

```diff
@@ -47,17 +71,6 @@ static const component_functions_t _component_functions = {
 
 /********************************** Create Instance **********************************/
 
-static void _continue_callback(component_t* component)
-{
-    data_t* data = (data_t*)component->parent->data;
-    if (!data->insert || sd_card_inserted()) {
```

Comment:
For removing the card, the continue button served to exit the workflow without an error. You changed it to show a cancel button only, which errors with UserAbort when pressed, which is a breaking change.

Please leave the semantics as it was - leave the continue button/callback and hide the cancel button/callback for removal. Otherwise we have to worry about how it will behave in the app :sweat_smile: